people.md

```diff
@@ -0,0 +1,13 @@
+##  Key persons in world of blockchain
+
+1. [Satoshi Nakamoto](/people/SatoshiNakamoto.md)
+2. [Vitalik Buterin](/people/VitalikButerin.md)
+3. [Gavin Wood](/people/GavinWood.md)
+4. [Dan Larrimer](/people/DanLarrimer.md)
+5. [Alexander Ivanov](/people/AlexanderIvanov.md)
+6. [Patrick Dai](/people/PatrickDai.md) 
+7. [Zooko Wilcox](/people/ZookoWilcox.md)
+8. [Reggie Middleton](/people/ReggieMiddleton.md) 
+9. [Brock Pierce](/people/BrockPierce.md)
+10.[Reeve Collins](/people/ReeveCollins.md)
```

Comment:
Here you break the numbered list.  Please, eyecheck your changes in github before submitting.